### PR TITLE
fix: set default text color in darkmode, and colors in storybook

### PR DESCRIPTION
**Pull request by alexandereneroth**
**Describe pull-request**  
- Makes the default text color of sdds white in dark mode
- Corrects darkmode colors in storybook

**Screenshots**  
Screenshot 2022-10-31 at 11 30 07
Screenshot 2022-10-31 at 11 29 50

**Additional context**  
Makes it so you can properly test your components in darkmode

### CODE REVIEW

**Comment by alexandereneroth** [comment]
@theJohnnyMe I made grey-50 the default background color, makes more sense to me now anyway 🙂

Didn't find a way to switch to darkmode without the storybook-dark-mode plugin unfortunately.
Seems like a decent maintainer anyway, he has a podcast about developer tooling 😃 https://www.youtube.com/channel/UCFsRlOn7gODgv6WUriLrzXg

**Comment by mJarsater** [comment]
Nice to see the Scania logo! Looks really good 😃

**Review by mJarsater — APPROVED** [review]

**Review by theJohnnyMe — APPROVED** [review]

**Comment by theJohnnyMe** [comment]
> @theJohnnyMe I made grey-50 the default background color, makes more sense to me now anyway 🙂
> 
> Didn't find a way to switch to darkmode without the storybook-dark-mode plugin unfortunately. Seems like a decent maintainer anyway, he has a podcast about developer tooling 😃 https://www.youtube.com/channel/UCFsRlOn7gODgv6WUriLrzXg

Great, thanks for checking on it anyway. Great work on common, scania light and scania dark theme ;)

File: tegel/.storybook/preview.js [hunk 1]
```diff
@@ -131,4 +141,28 @@ export const parameters = {
...
+  window.SDDS_DID_SUBSCRIBE_DARK_BG = true;
+  const darkModeBgColor = parameters.backgrounds.values.find(({ name }) => name === 'grey-958').value;
+  const lightModeBgColor = parameters.backgrounds.values.find(({ name }) => name === 'grey-50').value;
+  const channel = addons.getChannel();
+  channel.emit(UPDATE_GLOBALS, {
+    globals: { backgrounds: { value: lightModeBgColor } },
+  });
+  channel.on('DARK_MODE', isDarkMode => {
+    if ((isDarkMode && !window.SDDS_DARK_BG) || (!isDarkMode && window.SDDS_DARK_BG)) {
+      debugger;
```
Comment (alexandereneroth) [inline tegel/.storybook/preview.js:159]:
```suggestion
```

**Pull request merged**